tests/rules/experimental/rbe/network_enabled.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
same as above, license + short doc, same for the two other scripts below

Reply:
Done.